imgp

```diff
@@ -489,7 +497,7 @@ def parse_args(args=None, namespace=None):
                                        formatter_class=argparse.RawTextHelpFormatter)
     addarg = argparser.add_argument
     addarg('-x', '--res', nargs=1, metavar='res',
-           help='output resolution in HxV or percentage')
+           help='output resolution in HxV or percentage ("%%" suffix) or megapixel ("MP" suffix)')
```

Comment:
Singe "%" suffix for percentage.

Reply:
That's what I tried at first...
https://stackoverflow.com/questions/21168120/python-argparse-errors-with-in-help-string